docs/DesignDocs/InitTimeConfigurationInterface.md

```diff
@@ -83,28 +83,66 @@ typedef enum _oe_enclave_setting_type
      *  reflected in TEE-produced enclave identity evidence. Currently only
      *  supported by SGX Enclaves with KSS feature enabled.
      */
-    OE_SGX_CONFIGURATION_DATA = 0x78b5b41d
+    OE_SGX_ENCLAVE_CONFIG_DATA = 0x78b5b41d
 } oe_enclave_setting_type_t;
 
-typedef struct _oe_sgx_configuration_data
+typedef struct _oe_sgx_enclave_setting_config_data
 {
-    uint8_t config_id[8];
+    uint8_t config_id[64];
     uint16_t config_svn;
```

Comment:
here I think the abbreviation is ok since I think the intent is to literally match what is in an SGX processor specification for these field names (like SECS.CONFIGSVN is fine).